platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/FileStatisticAware.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.merkledb;
+
+import java.util.LongSummaryStatistics;
+
+/**
+ * This interface indicates that the data storage files in the implementing class can be compacted
```

Comment:
This interface doesn't look to be related to compactions at all, as stated by its JavaDoc

Reply:
Fixed.